File: Python/UPCMatricula/models/Alumno.py
class Alumno(object):
    def __init__(self, codigo, nombre, apellido):
        self.codigo = codigo
        self.nombre = nombre
        self.apellido = apellido

    def __str__(self):
        return self.nombre + " " + self.apellido

    def __repr__(self):
        return self.nombre + " " + self.apellido

File: PHP/UPCMatricula/models/Alumno.php
<?php

class Alumno {
    private $codigo;
    private $nombre;
    private $apellido;

    public function __construct($codigo, $nombre, $apellido) {
        $this->codigo = $codigo;
        $this->nombre = $nombre;
        $this->apellido = $apellido;
    }

    public function __toString() {
        return $this->nombre . "  " . $this->apellido;
    }
}
